src/Composer/Installers/MauticInstaller.php

```diff
@@ -4,6 +4,7 @@
 class MauticInstaller extends BaseInstaller
 {
     protected $locations = array(
+        'core'   => 'public/',
```

Comment:
Why is the directory named this way, and not, for example, `core`?

Reply:
that I am not sure, nor can I change that I think. It does work though :)